client/wfnews-war/src/main/angular/src/app/components/saved/saved.component.ts

```diff
@@ -86,9 +86,18 @@ export class SavedComponent implements OnInit {
         .subscribe((bans) => {
           this.savedLocations[outerIndex].bans = [];
           for (const innerIndex in bans?.features) {
-            const element = bans?.features[innerIndex];
-            this.savedLocations[outerIndex].bans.push(element);
-            this.cdr.markForCheck();
+            if (bans?.features.hasOwnProperty(innerIndex)) {
+              const element = bans?.features[innerIndex];
```

Comment:
You can remove the `?` here as the if check above already ensures the value exists

`hasOwnProperty` should not be executed on the object directly, but through the Object prototype

`Object.hasOwn(bans?.features, innerIndex)`